docs/src/accumulators.md

```diff
@@ -45,3 +45,36 @@ pop!(a, x)       # remove a key x from a, and return its current value
 merge(a, a2)     # return a new accumulator/counter that combines the
                  # values/counts in both a and a2
 ```
+
+
+merge is the multiset sum (Sometimes written ⊎) operation.
+
+## Use as a MultiSet
+
+An `Accumulator{T, <:Integer}` such as is returned by `counter`, is a [multiset](https://en.wikipedia.org/wiki/Multiset) or Bag, of objects of type `T`.
```

Comment:
If you don't care about the second parametric type then maybe write as: `Accumulator{T}`

Reply:
Ok, I think maybe you should write this as `Accumulator{T, V<:Integer}` because at first I thought you had a typo and meant `Accumulator{T<:Integer}`

Reply:
Alternatively `Accumulator{T, <:Integer} where T` would also indicate it was done on purpose to me